docs/reference/analysis/tokenfilters/keyword-marker-tokenfilter.asciidoc

```diff
@@ -1,137 +1,189 @@
 [[analysis-keyword-marker-tokenfilter]]
-=== Keyword Marker Token Filter
+=== Keyword marker token filter
+++++
+<titleabbrev>Keyword marker</titleabbrev>
+++++
 
-Protects words from being modified by stemmers. Must be placed before
-any stemming filters.
+Excludes specified tokens, called <<stemmer-keyword,keywords>>, from
+<<stemming,stemming>>.
```

Comment:
Maybe: 

Designates specific tokens as keywords so that they are not stemmed.

Reply:
Changed to the suggested text with b53322d6f4d.